src/uwtools/cli.py

```diff
@@ -298,12 +300,12 @@ def _dispatch_forecast_run(args: Namespace) -> bool:
 
     :param args: Parsed command-line args.
     """
-    forecast_class = uwtools.drivers.forecast.CLASSES[args.forecast_model]
+    forecast_class = uwtools.drivers.forecast.CLASSES[vars(args)[STR.model]]
```

Comment:
I think `forecast_model` was replaced with `model` somewhere along the line but this didn't get updated. Fix, and fixed corresponding unit test.